app/uk/gov/hmrc/merchandiseinbaggage/views/Layout.scala.html

```diff
@@ -50,7 +51,8 @@
     @maybeDeclarationType.map { declarationType =>
         <script>
                 dataLayer = [{
-                    'MIB_URL': '@declarationType.toString.toLowerCase' + window.location.pathname
+                    'MIB_URL': '@declarationType.toString.toLowerCase' + window.location.pathname,
+                    'MIB_JOURNEY_TYPE': '@request.session.get("journeyType").getOrElse("")'
```

Comment:
fyi, 
incase of earlier MIB_URL/declarationType we have passed the `maybeDeclarationType`
 from each view and then used it here, but now, for MIB_JOURNEY_TYPE , i am setting the `journeyType` in the session and using it here